decentraland/unity-renderer
# Fix: updated voice chat panel empty texts

## What does this PR change?

Fix #2958
Changed texts in voice chat panel when no players are connected

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/voice-chat-text
2. Open the voice chat panel when no players are connected
3. Verify that the texts match what was requested in #2958 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
